### ISSUE TITLE
[java] InvalidLogMessageFormat does not detect extra parameters when no placeholders

### ISSUE THREAD

**Issue by jogu** [pmd/pmd#2196]
**Affects PMD Version:** 6.20.0 

**Rule:** [InvalidLogMessageFormat](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#invalidlogmessageformat)

**Description:**

If the format string has no placeholders, PMD does not detect extra parameters.

**Code Sample demonstrating the issue:**

```
logger.warn("foo {}", "flibble", "moo", "blah", "blah"); // PMD flags this
logger.warn("foo", "flibble", "moo", "blah", "blah"); // PMD doesn't flag this
```

**Running PMD through:** *Maven*

**Comment by Fernal73** [comment]
https://github.com/pmd/pmd/issues/2187
Added similar issue with request for new rule .
